ckanext/unaids/dataset_transfer/logic.py

```diff
@@ -71,7 +71,7 @@ def send_dataset_transfer_emails(dataset_id, recipient_org_id):
     emails_succeeded = []
     for user in users_to_email:
         try:
-            subject = '{} {}'.format(dataset['title'], _('Dataset Transfer'))
+            subject = dataset['title'] + ' ' + _('Dataset Transfer')
```

Comment:
I don't think it's a good idea on how to do it. In python2 unfortunately you need to explicitly for unicode space for strings:
```
 subject = u'{} {}'.format(dataset['title'], _('Dataset Transfer'))
 ```
 Should do just enough. The subject will be properly encoded later down the line with `ckan/ckan/lib/mailer.py`:
 ```
    subject = Header(subject.encode('utf-8'), 'utf-8')
 ````